gax/backoff_policy.h

```diff
@@ -69,18 +69,16 @@ class BackoffPolicy {
  * This policy implements the truncated exponential backoff policy for
  * retrying operations. After a request fails, and subject to a separate
  * retry policy, the client library will wait for an initial delay before
- * trying again. If the second attempt fails the delay time is increased by a factor
- * of 2. The delay time growth stops at a maximum delay wait time.
- * The policy also randomizes the delay each time, to avoid
- * [thundering herd
+ * trying again. If the second attempt fails the delay time is increased by a
+ * factor of 2. The delay time growth stops at a maximum delay wait time. The
+ * policy also randomizes the delay each time, to avoid [thundering herd
  * problem](https://en.wikipedia.org/wiki/Thundering_herd_problem).
  *
  * Note: The random number generator used when calculating backoff time in
- *       OnCompletion is lazily created. Furthermore, GeneratorFactory::Generator
- *       may touch mutable shared state.
- *       As a result, OnCompletion is NOT thread-safe.
- *       All other methods, including clone, are thread-safe.
- *       ExponentialBackoffPolicy is therefore thread-compatible.
+ *       OnCompletion is lazily created. Furthermore,
+ * GeneratorFactory::Generator may touch mutable shared state. As a result,
```

Comment:
This is cruft from when I yanked the GeneratorFactory template. All sentences after the one about lazy generator creation no longer apply.

Reply:
Done